docker/Dockerfile_commcarehq_base

```diff
@@ -42,3 +44,5 @@ RUN pip install \
     ipython \
     --user --upgrade \
     && rm -rf /root/.cache/pip
+
+EXPOSE 8000
```

Comment:
Fair enough. I think I put it there because I was confused about why bootstrap started the server, but did not open any ports. Turns out it's because of the difference between docker-compose run and up.